src/app/_components/Card.tsx

```diff
@@ -21,21 +22,42 @@ type CardProps = {
   as?: React.ElementType
 }
 
-const borderStyles = {
-  'brand-300': 'border-brand-300',
-  'brand-400': 'border-brand-400',
-  'brand-500': 'border-brand-500',
-  'brand-600': 'border-brand-600',
-}
+type SpacingValue = keyof typeof theme.spacing
+type BreakpointValue = keyof typeof theme.screens
+
+type LeftProperty = `left-${SpacingValue}`
+type BottomProperty = `bottom-${SpacingValue}`
+
+type ResponsiveLeftProperty = `${BreakpointValue}:${LeftProperty}`
+type ResponsiveBottomProperty = `${BreakpointValue}:${BottomProperty}`
+
+type LinkProps = {
+  left?: LeftProperty | [LeftProperty, ResponsiveLeftProperty]
+  bottom?: BottomProperty | [BottomProperty, ResponsiveBottomProperty]
```

Comment:
maybe we could limit a number of options we can pass to the prop? For example, atm we only use 3 values for the left, and since (I assume) we won't have much variations we can limit to those we have until we need some other (to reduce the cognitive load of calculating).

Love the solution btw ❤️

Reply:
Do you mean replacing the current type declaration with hard-coded values (`left-36` and `left-4 sm:left-40`)? It's true that this config is a bit overkill given we only use it for a handful of values. So maybe your idea is simpler.

The `bottom` prop might be unnecessary for now since the CTA will always be 16px (4) from the bottom regardless of the layout. I added it for consistency, but maybe we remove it.